public/embed-widget.js

```diff
@@ -77,17 +77,29 @@ function initWidget() {
       var locale = (widget.dataset||{}).locale || "en"
 
       var urlMapping = {
-        "national-target-analyzer": "/national-targets/analyzer"
+        "national-target-analyzer": "/national-targets/analyzer",
+        "url"                     : 'raw'
       }
       var type = getAttributeValue(widget, "type")
 
       if (urlMapping[type]) {
         var width = getAttributeValue(widget, "width") || "100%"
         var height = getAttributeValue(widget, "height") || "500"
 
-        var iframeSrc = `${origin}/${locale}`
+        var iframeSrc = `${origin}/${locale}`;
+        if(urlMapping[type] == 'raw'){
 
-        iframeSrc += `${urlMapping[type]}?embed=true`
+          var rawUrl = getAttributeValue(widget, "url");
+          var parsedRawUrl = new URL(rawUrl);
+
+          if(!/ort\.cbd\.int$/.test(parsedRawUrl.hostname) && !/ort\.cbddev\.xyz$/.test(parsedRawUrl.hostname)){
```

Comment:
Shouldn't the _ort.cbd.int_ and _ort.cbddev.xyz_ pattern be in a variable?

Reply:
its fine here because only these two domain will support embedding